alphadia/workflow/base.py

```diff
@@ -95,8 +95,22 @@ def load(
             self._calibration_manager.disable_mobility_calibration()
 
         # initialize the optimization manager
+        optimization_manager_config = {
+            "ms1_error": self.config["search_initial"]["initial_ms1_tolerance"],
+            "ms2_error": self.config["search_initial"]["initial_ms2_tolerance"],
+            "rt_error": self.config["search_initial"]["initial_rt_tolerance"],
+            "mobility_error": self.config["search_initial"][
+                "initial_mobility_tolerance"
+            ],
+            "column_type": "library",
+            "num_candidates": self.config["search_initial"]["initial_num_candidates"],
+            "classifier_version": -1,
+            "fwhm_rt": self.config["optimization_manager"]["fwhm_rt"],
+            "fwhm_mobility": self.config["optimization_manager"]["fwhm_mobility"],
+            "score_cutoff": self.config["optimization_manager"]["score_cutoff"],
+        }
         self._optimization_manager = manager.OptimizationManager(
-            self.config["optimization_manager"],
+            optimization_manager_config,
```

Comment:
this seems to me like `OptimizationManager`-specific logic that is outside of `OptimizationManager` :-)
suggestion:
pass the `self.config` to the `OptimizationManager` and do this mapping during initialization. Also, I suggest separating between constant values (probable `fwhm_rt` does not change?) and dynamic values (e.g. classifier_version)

Reply:
Yes, that is true. I'll change it. 

I think everything in the optimization manager should change – fwhm_rt for instance is updated during calibration.